static/elements/chromedash-featurelist.html

```diff
@@ -130,38 +178,72 @@
-            this.filtered = this._filterOnOwner(ownerMatch[1]);
-          } else if (categoryMatch) {
-            this.filtered = this._filterOnCategory(categoryMatch[1]);
-          } else if (operatorMatch) {
-            this.filtered = this._filterOnOperation(operatorMatch[1], operatorMatch[2]);
+          var val = ' foo "bar baz" quz " quuz zzz"';
+          var regExp = /"([^"]*)"/g;
+          var start = 0;
+          var parts = [];
+          while ((match = regExp.exec(val)) !== null) {
+            if (start - (regExp.lastIndex - match[0].length) > 0) {
+              parts.push(val.substring(start, regExp.lastIndex -
+                                       match[0].length));
+            }
+            parts.push(match[1]);
+          }
+          if (start - (regExp.lastIndex - match[0].length) > 0) {
+            parts.push(val.substring(start, regExp.lastIndex -
+                                     match[0].length));
+          }
+
+          // Match words separated by whitespace and/or ":".
+          regExp = /([^:<>= \f\n\r\t\v\u00a0\u1680\u180e\u2000-\u200a\u2028\u2029\u202f\u205f\u3000\ufeff]+)([:<>= \f\n\r\t\v\u00a0\u1680\u180e\u2000-\u200a\u2028\u2029\u202f\u205f\u3000\ufeff]+)?/g;
+          var match;
+          var reMatches = [];
+          while (match = regExp.exec(val) !== null) {
+            reMatches.push(match);
+          }
+
+          // Accumulate keyword and property queries.
+          var propertyQueries = [];
+          var operatorQueries = [];
+          var keywordQueries = [];
+          var i;
+          for (i = 0; i < reMatches.length; i++) {
+            match = reMatches[i];
+            if (match[2] && match[2].match(':') !== null &&
+                i < reMatches.length - 1) {
+              propertyQueries.push([match[1], reMatches[i + 1][1]]);
+              i++;
+            } else if (match[2] && match[2].match(/[<>=]/) !== null &&
+                       i < reMatches.length - 1) {
+              operatorQueries.push([match[1], match[2], reMatches[i + 1][1]]);
+              i++;
+            } else {
+              keywordQueries.push(match[1]);
+            }
+          }
+
+          var filters = propertyQueries.map(
+            this._getPropertyFilter.bind(this)).concat(
+              operatorQueries.map(
+                this._getOperatorFilter.bind(this))).concat(
+                  keywordQueries.map(
+                    this._getKeywordFilter.bind(this)));
+
+          if (filters.length === 0) {
+            this.filtered = this.features;
           } else {
-            // Only one of the platforms needs to match.
-            // Do a general search over misc fields.
-            // Remove "=" prefix so "in development"/"proposed" version queries
-            // return results.
-            if (val.indexOf('=') == 0) {
-              val = val.substring(1);
+            var results = this.features;
+            var j;
+            for (i = 0; i < filters.length; i++) {
```

Comment:
The `var j` is wrong; possibly left over from a different implementation. See my previous note on the non-var-iness.